tomviz/vtkActiveScalarsProducer.cxx

```diff
@@ -0,0 +1,78 @@
+#include "vtkActiveScalarsProducer.h"
+
+#include <vtkExecutive.h>
+#include <vtkGarbageCollector.h>
+#include <vtkImageData.h>
+#include <vtkObjectFactory.h>
+#include <vtkPointData.h>
+
+vtkStandardNewMacro(vtkActiveScalarsProducer);
+
+vtkActiveScalarsProducer::vtkActiveScalarsProducer()
+{
+  this->OriginalOutput = nullptr;
```

Comment:
We should use C++11 and assign nullptr directly at declaration in the header for both of these.